src/projetos/CadastroProjetosPage.tsx

```
@@ -131,35 +136,47 @@ export function CadastroProjetoPage() {
             justificativa: projeto.justificativa,
             dataInicio: new Date(projeto.dataInicio),
             dataFim: new Date(projeto.dataFim),
-            restricaoPublico: projeto.restricaoPublico,
             vinculadoDisciplina: projeto.vinculadoDisciplina,
-            equipeExecutora:
-              projeto.equipeExecutora?.map((p: Servidor) => p.email) || [],
-            cargaHoraria: projeto.cargaHoraria,
-            qtdeVagas: projeto.qtdeVagas,
+            restricaoPublico: projeto.restricaoPublico || '',
+            publicoAlvo: projeto.publicoAlvo,
+            status: "EM_ANDAMENTO",
```

ESLint

[error] Replace `"EM_ANDAMENTO"` with `'EM_ANDAMENTO'`

(prettier/prettier)